### DISCUSSION TITLE
Bluetooth phone sensors

### DISCUSSION THREAD

**Discussion by PacmanForever** [OpenTracksApp/OpenTracks#2027] [General]
Hello again, I want to configure the sensors of the mobile phone in the OT (Heart rate and barometer), but it does not show me any bluetooth sensor in the configuration. I have a Huawei GT2 watch connected to a OnePlus 10T mobile, but OT does not detect any sensor.

Am I doing something wrong?

Is there a list of compatible mobile phones and devices?

Thanks

**Comment by dennisguse** [comment]
OpenTracks supports all heartrate sensors that announce that they are heartrate sensors.
If the GT 2 doesn't do that you can disable the filter and then the watch will be selectable as heartrate source (as all other BLE devices in your vincinity).
However, the GT 2 also needs to support the standard approach to provide the actual heartrate.
Sadly, often smartwatches use some proprietary protocol, because "who needs standards, right?"

And no you are (very likely) not doing anything wrong.

Here is a list of known to work sensors (but it is short): https://github.com/OpenTracksApp/OpenTracks/blob/main/README_TESTED_SENSORS.md#0x180d-heartrate-service

PS/ the problem is not your phone ;)

**Reply by PacmanForever** [reply]
ok, thanks
